modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc

```diff
@@ -340,7 +340,7 @@ State:                      ready
 Private:                    No
 Created:                    Oct  1 2021 08:12:25 UTC
 Details Page:               https://console.redhat.com/openshift/details/s/<subscription_id>
-OIDC Endpoint URL:          https://rh-oidc.s3.<aws_region>.amazonaws.com/<cluster_id>
+OIDC Endpoint URL:           https://d3gt1gce2zmg3d.cloudfront.net/<cluster_id>
```

Comment:
Users will see the production URL
```suggestion
OIDC Endpoint URL:           https://dvbwgdztaeq9o.cloudfront.net/<cluster_id>
```
Reference: https://issues.redhat.com/browse/APPSRE-7296